stant-album/Request.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Request.swift
```

Comment:
What are you requesting? hehe.  Always put legible names on classes.  Is it a "util" class?

Reply:
Done.